lib/cmd/common/build.js

```diff
@@ -14,7 +14,9 @@ build.usage = "\nfhc build app=<app-id> destination=<destination> version=<versi
   + i18n._("\n'keypass' and 'certpass' only needed for 'release' builds")
   + i18n._("\n'provisioning' is only optional for iphone or ipad builds");
 
-build.usage_ngui = "\nfhc build project=<project-id> app=<app-id> cloud_app=<cloud-app-id> tag=<tag> destination=<destination> version=<version> config=<config> keypass=<private-key-password> certpass=<certificate-password> download=<true|false> provisioning=<path-to-provisioning-profile> cordova_version=<cordova version>  branch=<branch name> commit=<commit hash>"
+build.usage_ngui = "\nfhc build project=<project-id> app=<app-id> cloud_app=<cloud-app-id> tag=<tag> environment=<environment> destination=<destination> version=<version> config=<config> keypass=<private-key-password> certpass=<certificate-password> download=<true|false> provisioning=<path-to-provisioning-profile> cordova_version=<cordova version>  branch=<branch name> commit=<commit hash>"
+  + i18n._("\nwhere <tag name> is the name of a connection tag for the cloud app, must be in Semantic Version format, e.g. 0.0.1.")
+  + i18n._("\nwhere <environment name> is the name of a target environment, e.g. dev")
```

Comment:
strictly speaking, I believe it is the environment id that is required. Not the environment name. Can we just change it here to avoid confusion.